src/integrations/pulsar/consumer.cpp

```diff
@@ -14,6 +14,7 @@
 #include <fmt/format.h>
 #include <pulsar/Client.h>
 #include <pulsar/InitialPosition.h>
+#include <algorithm>
 
 #include <chrono>
 #include <thread>
```

Comment:
```suggestion
#include <algorithm>
#include <chrono>
#include <thread>

#include <fmt/format.h>
#include <pulsar/Client.h>
#include <pulsar/InitialPosition.h>
```
Sorry, I haven't noticed this before, but it is quite straightforward.